README.md

```diff
@@ -273,6 +273,18 @@ module.exports = {
 
 Any `node_modules` passed into this option will be compiled through webpack as if they are part of your app.
 
+### Polyfills
+
+Sometime you might need to polyfill a new object or method in browsers that don’t support that object or method, and you might also want polyfill is loaded and executed as part of entry point without your files needing to know about it.
+```js
+module.exports = {
+  polyfills: ['core-js/modules/es6.symbol', 'regenerator-runtime/runtime']
+}
+```
+
+Note: From a performance point of view, you should only send polyfills to browsers that need them.
```

Comment:
I feel like this line can be removed, especially since sku doesn't help you achieve this very well :)